config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -1,7 +1,50 @@
 apiVersion: dataplane.openstack.org/v1beta1
 kind: OpenStackDataPlane
 metadata:
-  name: openstackdataplane-sample
+  name: openstack-edpm
 spec:
-  dataPlaneRoles:
-    - openstackdataplanerole-sample
+  deployStrategy:
+      deploy: true
```

Comment:
can we set this to false here to match the other samples?

Reply:
yes